src/main/java/org/mule/runtime/dsl/api/xml/parser/XmlApplicationParser.java

```diff
@@ -44,6 +44,15 @@
   public static final String LINE_NUMBER = "LINE_NUMBER";
   public static final String CONFIG_FILE_NAME = "CONFIG_FILE_NAME";
   public static final String IS_CDATA = "IS_CDATA";
+  public static final String OPENING_TAG_START_LINE = "OPENING_TAG_START_LINE";
```

Comment:
I should better move this to a separate Interface/Enum, right?
Including `IS_CDATA` probably. However I could not remove that one from here without breaking backwards.

Reply:
it would be better to move these to their own interface, yes. You still have to keep the IS_CDATA here as a delegate for backwards compatibility